src/par_iter/mod.rs

```diff
@@ -646,6 +650,14 @@ pub trait IndexedParallelIterator: ExactParallelIterator {
         Enumerate::new(self)
     }
 
+    fn skip(self, n: usize) -> Skip<Self> {
+        Skip::new(self, n)
+    }
+
+    fn take(self, n: usize) -> Take<Self> {
```

Comment:
These should have at least a little documentation, even if there's not much to say. :)